main/end-to-end-tests/cypress/integration/workspacetypes.spec.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
This file should be named `workspace-types.spec.js`

Reply:
Got it. Will make the change